syft/frameworks/torch/he/fv/util/operations.py

```diff
@@ -111,22 +152,26 @@ def poly_mul_mod(op1, op2, coeff_mod):
         A list with polynomial coefficients.
     """
 
-    # For non same size polynomails we have to shift the polynomials because numpy consider right
+    # For non same size polynomials we have to shift the polynomials because numpy consider right
     # side as lower order of polynomial and we consider right side as heigher order.
-    if len(op1) != len(op2):
-        if len(op1) > len(op2):
-            op2 = op2 + [0] * (len(op1) - len(op2))
-        else:
-            op1 = op1 + [0] * (len(op2) - len(op1))
+    if len(op1) != poly_mod:
```

Comment:
Same question about the poly_mod format.

Reply:
https://github.com/OpenMined/PySyft/pull/3711#discussion_r446906200